py/specter/util/util.py

```diff
@@ -242,4 +242,135 @@ def legval_numba(x, c):
         c0 = c[-i] - (c1*(nd - 1))*nd_inv
         c1 = tmp + (c1*x*(2*nd - 1))*nd_inv
     return c0 + c1*x
+  
+
+@numba.jit(nopython=True, cache=False)
+def custom_hermitenorm(n, u):
```

Comment:
Please add a docstring that defines `n` and `u` and describes that this is functionally equivalent to
```python
fn = scipy.special.hermitenorm(n)
return fn(u)
```
(if indeed that is the case).  Include any caveats about how this is different from the scipy version (e.g. requiring array input instead of accepting arrays or scalar, etc.).

Reply:
Docstring added. Note about supporting arrays but not scalars also included.